src/components/Adresses.vue

```diff
@@ -0,0 +1,179 @@
+        </span>
+      </multiselect>
+    </div>
+
+    <p class="text-warning" v-if="(cities && cities.length === 0) || (locales && locales.length === 0)">😔 Infelizmente não encontramos suporte em sua cidade, mas não acabou! Ligue para 188, é de graça!</p>
+
+    <div class="locales">
+      <div class="locales-card" v-for="item in locales">
+        <p><strong>Telefone</strong>: {{ item.phone }}</p>
+        <p><strong>Horário de Atendimento</strong>: {{ item.time }}</p>
+        <p><strong>Endereço</strong>: {{ item.address }}</p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+ import Multiselect from 'vue-multiselect'
+
+  export default {
+    name: "AdressesList",
+    components: { Multiselect },
+    data () {
+      return {
+        ufValue: null,
+        cityValue: null,
+        cities: null,
+        locales: null,
+        states: [
+          {value: "AC", label: "Acre"},
+          {value: "AL", label: "Alagoas"},
+          {value: "AM", label: "Amazonas"},
+          {value: "AP", label: "Amapá"},
+          {value: "BA", label: "Bahia"},
+          {value: "CE", label: "Ceará"},
+          {value: "DF", label: "Distrito Federal"},
+          {value: "ES", label: "Espírito Santo"},
+          {value: "GO", label: "Goiás"},
+          {value: "MA", label: "Maranhão"},
+          {value: "MG", label: "Minas Gerais"},
+          {value: "MS", label: "Mato Grosso do Sul"},
+          {value: "MT", label: "Mato Grosso"},
+          {value: "PA", label: "Pará"},
+          {value: "PB", label: "Paraíba"},
+          {value: "PE", label: "Pernambuco"},
+          {value: "PI", label: "Piauí"},
+          {value: "PR", label: "Paraná"},
+          {value: "RJ", label: "Rio de Janeiro"},
+          {value: "RN", label: "Rio Grande do Norte"},
+          {value: "RO", label: "Rondônia"},
+          {value: "RR", label: "Roraima"},
+          {value: "RS", label: "Rio Grande do Sul"},
+          {value: "SC", label: "Santa Catarina"},
+          {value: "SE", label: "Sergipe"},
+          {value: "SP", label: "São Paulo"},
+          {value: "TO", label: "Tocantins"},
+        ]
+      }
+    },
+  methods: {
+    fetchCitiesThatHasCVV({value: uf}) {
+        /* TODO: Colocar em contantes? */
+        var self = this;
+
+        self.cityValue = null;
+        self.cities = null;
+        self.locales = null;
+
+        const proxy = `https://cors-anywhere.herokuapp.com/<url>`
+        const citiesURL = `https://www.cvv.org.br/search/cities?stateId=${uf}`
+
+        const url = proxy.replace('<url>', citiesURL)
+
+        console.log(url)
+
+        fetch(url).then(res => res.json()).then(res => self.cities = res)
+    },
+    fetchCVVsInACity({ id }) {
+         /* TODO: Colocar em contantes? */
+        var self = this;
+
+        const proxy = `https://cors-anywhere.herokuapp.com/<url>`
+        const cvvsURL = `https://www.cvv.org.br/search/places?city_id=${ id }`
```

Comment:
Usar um único padrão para fazer bind em string. Usar `replace`.

```suggestion
        const proxy = `https://cors-anywhere.herokuapp.com/<url>`
        const placesURL = `https://www.cvv.org.br/search/places?city_id=<city_id>`
        
        const proxyPlacesURL = proxy.replace('<url>', placesURL).replace('<city_id>', id)
        
        fetch(proxyPlacesURL).....
```

Reply:
Achei demais!